Modules/Relationships/test/Relationships.Domain.Tests/Tests/Aggregates/Relationships/RelationshipReactivationRequestTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Backbone.BuildingBlocks.Domain;
+using Backbone.Modules.Relationships.Domain.Aggregates.Relationships;
+using Backbone.Modules.Relationships.Domain.Tests.Extensions;
+using Backbone.Tooling;
+using FluentAssertions;
+using Xunit;
+using static Backbone.Modules.Relationships.Domain.Tests.TestHelpers.TestData;
+
+namespace Backbone.Modules.Relationships.Domain.Tests.Tests.Aggregates.Relationships;
+public class RelationshipReactivationRequestTests
+{
+    [Fact]
+    public void Relationship_reactivation_request_creates_an_audit_log_entry()
```

Comment:
Test names should be natural language as far as possible
```suggestion
    public void Requesting_reactivation_creates_an_audit_log_entry()
```

Reply:
Changes made.